extensions/wikia/Flags/Flags.i18n.php

```diff
@@ -10,7 +10,7 @@
  */
 $messages['en'] = [
 	'flags-description' => 'Portable flags',
-	'flags-edit-form-more-info' => 'More info',
+	'flags-edit-form-more-info' => 'More info >',
```

Comment:
&rsaquo; renders poorly - drops to the bottom instead of vertical middle. Leaving > for now.